src/Symfony/Component/DependencyInjection/LazyProxy/PhpDumper/DumperInterface.php

```diff
@@ -38,7 +38,7 @@ public function isProxyCandidate(Definition $definition);
      *
      * @return string
      */
-    public function getProxyFactoryCode(Definition $definition, $id, $methodName);
+    public function getProxyFactoryCode(Definition $definition, $id, $methodName = null);
```

Comment:
sure, that's even what we planned for in 3.4, see
https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/DependencyInjection/LazyProxy/PhpDumper/DumperInterface.php#L41